jenkins/init-aptly-repo.sh

```diff
@@ -73,4 +73,4 @@ aptly_nginx_config=${aptly_nginx_config//'{vm_fqdn}'/${vm_fqdn}}
 echo "${aptly_nginx_config}" | sudo tee -a /etc/nginx/sites-enabled/default > /dev/null
 
 #restart nginx
-sudo service nginx restart
```

Comment:
Keep the line ending unchanged, same for every file in the pull request

Reply:
Any reason to remove the nginx restart cmd?